src/OpenLoco/src/OpenLoco.cpp

```diff
@@ -124,21 +124,12 @@ namespace OpenLoco
     // 0x004BE621
     [[noreturn]] void exitWithError(StringId eax, StringId ebx)
```

Comment:
Could you change these parameters to read something like titleStringId, messageStringId?